src/mpid/ch4/netmod/ofi/ofi_init.c

```diff
@@ -679,15 +687,13 @@ int MPIDI_OFI_init_local(int *tag_bits)
 
     MPIDI_OFI_global.num_vnis = num_vnis;
 
-    /* Create MPIDI_OFI_global.ctx[0] first  */
-    mpi_errno = create_vni_context(0);
-    MPIR_ERR_CHECK(mpi_errno);
-
-    /* Creating the additional vni contexts.
+    /* Creating the vni contexts.
      * This code maybe moved to a later stage */
-    for (int i = 1; i < MPIDI_OFI_global.num_vnis; i++) {
-        mpi_errno = create_vni_context(i);
-        MPIR_ERR_CHECK(mpi_errno);
+    for (int i = 0; i < MPIDI_OFI_global.num_vnis; i++) {
+        for (int j = 0; j < MPIDI_OFI_global.num_nics; j++) {
```

Comment:
Swap the order of loops, first `nic` then `vni`? Maybe it doesn't matter, but my mind is stuck with first nic then each nic with multiple vnis, all because your first commit said so 😄

Reply:
That's a mistake in the commit message. Sorry about that. I think in the internal version of this, we didn't already have this version of the multi-VCI code so the ordering was swapped. I'd rather change the commit message than change all of the code. :)

Reply:
I see what you're saying now. I can understand why this might seem a little bit backward. At the same time, I'm lazy and don't want to go change 53 places where I use `MPIDI_OFI_get_ctx_index` to swap the order of the arguments. 😄 

I did at least fix a typo in that commit message.